### PR TITLE
cards

### PR DESCRIPTION
**Pull request by VladyslavKolisnyk**
DEMO LINK https://VladyslavKolisnyk.github.io/layout_product-cards
TEST REPORT LINK https://VladyslavKolisnyk.github.io/layout_product-cards/report/html_report

### CODE REVIEW

File: src/css/card.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  max-width: 200px;
```
Comment (polosanya) [inline src/css/card.css:5]:
```suggestion
  width: 200px;
```
I think it's better to make the width fixed

File: src/css/card.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px ;
+}
+
+.card__title {
+  font-family: "Roboto", sans-serif;
```
Comment (polosanya) [inline src/css/card.css:19]: You can set the font for the whole page and inherit where needed

File: src/css/index.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+@import url(./normaliz.css);
```
Comment (polosanya) [inline src/css/index.css:1]:
```suggestion
@import url(./normalize.css);
```

File: src/css/index.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+@import url(./stars.css);
+@import url(./card.css);
+@import url(./varible.css);
```
Comment (polosanya) [inline src/css/index.css:4]:
```suggestion
@import url(./variables.css);
```

File: src/css/stars.css [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  background-position: center;
+  border-radius: 0.5px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
```
Comment (polosanya) [inline src/css/stars.css:19]: Styles should work for stars with other modifiers too
image

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,55 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__reviews">Reviews:5</div>
+      </div>
+      <div class="card__price">
+        <div class="card__price-title">Price:</div>
+        <div class="card__price-cost">$2,199</div>
+      </div>
```
Comment (polosanya) [inline src/index.html:45]: image

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,55 @@
...
+      <a
+        href="#"
+        class="card__link"
+        data-qa="hover"
+      >
+      Buy
+      </a>
```
Comment (polosanya) [inline src/index.html:52]:
```suggestion
      <a
        href="#"
        class="card__link"
        data-qa="hover"
      >
        Buy
      </a>
```

**Review by polosanya — CHANGES_REQUESTED** [review]
Good job! Let's improve your code a bit)
Pay attention to the task description and checklist

**Review by VitaliyBondarenko1982 — APPROVED** [review]

**Pull request closed without merging**